kde-misc/markdownpart/markdownpart24-24.08.2.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="Markdown viewer KPart"
+DESCRIPTION="The Markdown Viewer KPart allows KParts-using software to display files in Markdown \
+format in a rendered view.
+Extends: Ark icon Ark | Kate icon Kate | KDevelop icon KDevelop | Konqueror icon Konqueror | \
+Krusader icon Krusader"
```

Comment:
These "<app name> icon" parts look like the alt text of actual icons. I think these should be removed and instead just the names listed. It is rather confusing this way.

Reply:
whoops, think you mentioned this before :) Will remove/adjust later